engine/flink/management/sample/src/main/scala/pl/touk/nussknacker/engine/management/sample/transformer/DynamicParametersSample.scala

```diff
@@ -0,0 +1,47 @@
+package pl.touk.nussknacker.engine.management.sample.transformer
```

Comment:
Hmm, why sample.. I think we should do Two traits: DynamicParametersMixin and DynamicParametersSample. After that DynamicParametersSource should extend by DynamicParametersMixin

Reply:
Ok, changed name in previous PR